fuel-vm/src/storage.rs

```diff
@@ -160,5 +160,7 @@ macro_rules! double_key {
     };
 }
 
+pub(crate) use double_key;
```

Comment:
You need to export macro=)

Reply:
Oh that is only within the same crate my bad